Back End/MemorizeWords-UnitTest/UnitTest.cs

```diff
@@ -82,8 +84,28 @@ public void Answer_ShouldAddRecord()
         private static DbContextOptions<EFCoreDbContext> GetDbContextOptions()
         {
             return new DbContextOptionsBuilder<EFCoreDbContext>()
-                            .UseInMemoryDatabase(databaseName: "TestDatabase")
-                            .Options;
+                .UseInMemoryDatabase(databaseName: "TestDatabase")
+                .Options;
+        }
+
+        [Fact]
+        public void IS_DEFAULT_TEST_SHOULD_DONE()
```

Comment:
this method looks great.

Can you move this method to new UnitTest class?

Create a new Folder called Utility and Create a class called ValidationUtilityTests.cs and put this method under it 

Great!

Reply:
Can you add one more test for class type?

Reply:
Done.